internal/federationdomain/endpointsmanager/manager.go

```diff
@@ -108,6 +119,7 @@ func (m *Manager) SetFederationDomains(federationDomains ...*federationdomainpro
 			tokenHMACKeyGetter,
 			nil,
 			timeoutsConfiguration,
+			m.auditLogger,
```

Comment:
All the handlers created in this `SetFederationDomains` function should probably log their request parameters. These could probably get done in a follow-up PR.

Reply:
The health and the three discovery endpoints completely ignore their parameters, so there's no need to audit log them.

The Choose IDP page does a lightweight validation of the params, but then just sends them back to the authorize endpoint on a redirect (with the "pinniped_idp_name" param added). So in this flow you would see the params audited at the authorize endpoint, then you would see them audited again (with one extra param) at the second call to the authorize endpoint. So there's no need to audit log the params for the Choose IDP page either.

All the other Supervisor FederationDomain endpoints audit log their parameters on this branch.